huntsman/tests/test_camera.py

```diff
@@ -311,7 +313,7 @@ def test_autofocus_coarse(camera, patterns):
         pytest.skip("Camera does not have a focuser")
     try:
         autofocus_event = camera.autofocus(coarse=True)
-        autofocus_event.wait()
+        autofocus_event.wait(timeout=30)
         assert len(glob.glob(patterns['final'])) == 1
     finally:
         shutil.rmtree(patterns['base'])
```

Comment:
Something weird going on where this cleanup code seems to be deleting all the directories _after_ the next test has started to take an image, causing that test to hang. Oddly this doesn't happen in `develop`, only in this branch.

I tried just commenting out the `rmtree` line but then the following test failed because the glob pattern finds too many 'final' files (the one from that test and the one from this test).